config/routes.rb

```diff
@@ -4,9 +4,10 @@
   get '/applications/new', to: 'applications#new'
   get '/applications/:id', to: 'applications#show'
   post '/applications', to: 'applications#create'
+  patch '/applications/:id', to: 'applications#update'
 
   get '/pet_applications/new', to: 'pet_applications#new'
-  post 'applications/:application_id/pet_applications', to: 'pet_applications#create'
+  post '/applications/:application_id/pet_applications', to: 'pet_applications#create'
```

Comment:
Nice work with the paths. Maybe we will end of re-organizing them later like Kat mentioned